.gitignore

```diff
@@ -21,3 +21,10 @@ db.php
 .Trashes
 ehthumbs.db
 Thumbs.db
+
+# Local PHPCS configuration files
+.phpcs.xml
+phpcs.xml
+
+# Composer (repo root only)
+/vendor/
```

Comment:
If we didn't include the leading `/`, then it would stop `vendor` directories in themes and plugins from being committed as well.